modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/EditAPIApplication.testcase

```diff
@@ -29,6 +29,245 @@ definition {
 		}
 	}
 
+	@priority = 4
+	test CanEditSchemaNameAndDescriptionInPublishedAPIApplication {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
```

Comment:
In headless qa team, we are now don't need to set `virtual.instance` and `test.run.type` by default unless there is some problems run on a virtual instance. So please help remove these two properties
```
property test.liferay.virtual.instance = "false";
		property test.run.type = "single";
```

Reply:
Done! thanks.